268.missing-number.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * https://leetcode.com/problems/missing-number/description/
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var missingNumber = function (nums) {
+  let sortedNumsArr = nums.sort((a, b) => a - b);
+  let result = 0;
+
+  for (let i = 0; i <= sortedNumsArr.length; i++) {
+    if (sortedNumsArr[i] !== i) {
+      if (sortedNumsArr[i] > i || sortedNumsArr[i] === undefined) {
+        result = i;
+        return result;
+      }
+    }
+  }
```

Comment:
```suggestion
  const sortedNumsArr = nums.sort((a, b) => a - b);
    
    for (let i = 0; i <= sortedNumsArr.length; i++) {
      if (sortedNumsArr[i] !== i|| sortedNumsArr[i] > i || sortedNumsArr[i] === undefined) {
        return i;
      }
    }
```
You can organize the code like this